ui/desktop/src/App.tsx

```diff
@@ -74,6 +103,125 @@ export default function App() {
     return `${cmd} ${args.join(' ')}`.trim();
   }
 
+  const disableAllStoredExtensions = () => {
```

Comment:
i think this changes pretty significantly for settingsv2, i'm thinking we might want to get a head start on that if that's planning to be released this week